test/tests/bcs/ad_matched_value_bc/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
```

Comment:
As for the other PR you may want to add a `PetscJacobianTest`

Reply:
👍